core/infra/config/peer_service_with_json.hpp

```diff
@@ -17,28 +17,29 @@ limitations under the License.
 #ifndef PEER_SERVICE_WITH_JSON_HPP
 #define PEER_SERVICE_WITH_JSON_HPP
 
-#include "../../service/peer_service.hpp"
-#include "iroha_config.hpp"
 #include <vector>
+#include <service/peer_service.hpp>
+#include "abstract_config_manager.hpp"
+
+using ACM = config::AbstractConfigManager;
```

Comment:
Avoid using file-wide type aliases if they are used only once or twice in the file (e.g. only for specifying a base class).